faker/providers/emoji/__init__.py

```diff
@@ -0,0 +1,4429 @@
+                ["U+1F587", "linked paperclips", ["link", "paperclip"]],
+                ["U+1F4CF", "straight ruler", ["ruler", "straight edge"]],
+                ["U+1F4D0", "triangular ruler", ["ruler", "set", "triangle"]],
+                ["U+2702", "scissors", ["cutting", "tool"]],
+                ["U+1F5C3", "card file box", ["box", "card", "file"]],
+                ["U+1F5C4", "file cabinet", ["cabinet", "file", "filing"]],
+                ["U+1F5D1", "wastebasket", []],
+            ],
+            "other-object": [
+                ["U+1F6AC", "cigarette", ["smoking"]],
+                ["U+26B0", "coffin", ["death"]],
+                ["U+26B1", "funeral urn", ["ashes", "death", "funeral", "urn"]],
+                ["U+1F5FF", "moai", ["face", "moyai", "statue"]],
+            ],
+            "phone": [
+                [
+                    "U+1F4F1",
+                    "mobile phone",
+                    ["cell", "mobile", "phone", "telephone"],
+                ],
+                [
+                    "U+1F4F2",
+                    "mobile phone with arrow",
+                    ["arrow", "cell", "mobile", "phone", "receive"],
+                ],
+                ["U+260E", "telephone", ["phone"]],
+                [
+                    "U+1F4DE",
+                    "telephone receiver",
+                    ["phone", "receiver", "telephone"],
+                ],
+                ["U+1F4DF", "pager", []],
+                ["U+1F4E0", "fax machine", ["fax"]],
+            ],
+            "science": [
+                ["U+2697", "alembic", ["chemistry", "tool"]],
+                [
+                    "U+1F9EA",
+                    "test tube",
+                    ["chemist", "chemistry", "experiment", "lab", "science"],
+                ],
+                [
+                    "U+1F9EB",
+                    "petri dish",
+                    ["bacteria", "biologist", "biology", "culture", "lab"],
+                ],
+                [
+                    "U+1F9EC",
+                    "dna",
+                    ["biologist", "evolution", "gene", "genetics", "life"],
+                ],
+                ["U+1F52C", "microscope", ["science", "tool"]],
+                ["U+1F52D", "telescope", ["science", "tool"]],
+                [
+                    "U+1F4E1",
+                    "satellite antenna",
+                    ["antenna", "dish", "satellite"],
+                ],
+            ],
+            "sound": [
+                [
+                    "U+1F507",
+                    "muted speaker",
+                    ["mute", "quiet", "silent", "speaker"],
+                ],
+                ["U+1F508", "speaker low volume", ["soft"]],
+                ["U+1F509", "speaker medium volume", ["medium"]],
+                ["U+1F50A", "speaker high volume", ["loud"]],
+                ["U+1F4E2", "loudspeaker", ["loud", "public address"]],
+                ["U+1F4
```

Comment:
Instead of creating the methods on `__new___`, would something in `__getattr__` work?